synapse/module_api/__init__.py

```diff
@@ -431,6 +433,14 @@ def public_room_list_manager(self) -> "PublicRoomListManager":
         """
         return self._public_room_list_manager
 
+    @property
+    def account_data_manager(self) -> "AccountDataManager":
```

Comment:
Any reason we don't just set the object as `account_data_manager` to start with? Just to show it really is public?

Reply:
I don't really know, but this is a convention through the rest of the module API. Perhaps it's to make it read-only? Happy to go with the simple approach if you think it's best; that's what I would have done had there not already been this pattern here.

Reply:
Following what else is done is fine too! 👍